docs/other/unity.md

```diff
@@ -16,47 +16,15 @@ Visual Studio Code can be a great companion to Unity for editing and debugging C
 
 Read on to find out how to configure Unity and your project to get the best possible experience.
 
-## Connecting Unity and VS Code
+## Setup VS Code as Unity Script Editor
 
-The easiest way to get going is to leverage a [Unity plug-in](https://github.com/dotBunny/VSCode/) maintained by [@Reapazor](https://twitter.com/reapazor).  This plug-in streamlines the integration process significantly by performing the following tasks:
+Simply open up *Unity Preferences*, *External Tools*, then browse **Visual Studio Code** executable as *External Script Editor*.
 
-1. Sets VS Code as the default editor - opening a script now opens it in VS Code
-2. Configures Unity to pass file and line numbers and reuse the existing window - so VS Code opens in the correct context
-3. Scrubs the Unity project file to ensure that OmniSharp can work with it - to get the best editing experience
-4. Configures VS Code to ignore certain Unity file types - removing clutter from the VS Code file explorer
-5. Configures a `launch.json` file with the correct debug port - to enable debugging. However, installing the [Debugger for Unity](https://marketplace.visualstudio.com/items/Unity.unity-debug) is recommended to get debugging support.
+![Unity Preferences](images/unity/Unity_Preferences_External_Script_Editor.gif)
 
-### Step 1: Download the plug-in code

```

Comment:
The plugin is not needed anymore, as this behaviour is supported in Unity by default.   the plugin author updated the description at https://github.com/dotBunny/VSCode/blob/master/README.md#requirements